<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="AUM" ignored="0" name="Login Test" tests="3" failures="1" timestamp="11 Apr 2020 13:22:42 GMT" time="0.169" errors="0">
  <testcase name="test1" time="0.025" classname="com.demo.nopcommerce.TesstNG.TestNGListenersDemo"/>
  <testcase name="test2" time="0.014" classname="com.demo.nopcommerce.TesstNG.TestNGListenersDemo">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.demo.nopcommerce.TesstNG.TestNGListenersDemo.test2(TestNGListenersDemo.java:22)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase name="test3" time="0.0" classname="com.demo.nopcommerce.TesstNG.TestNGListenersDemo">
    <skipped/>
  </testcase> <!-- test3 -->
</testsuite> <!-- Login Test -->
